web/ui/react-app/src/Panel.tsx

```diff
@@ -153,14 +154,29 @@ class Panel extends Component<PanelProps, PanelState> {
         throw new Error(json.error || 'invalid response JSON');
```

Comment:
When there's a query error, we should either hide the previously shown query stats or show stats for the errored query (like the old UI does, see http://demo.robustperception.io:9090/graph?g0.range_input=1h&g0.expr=-invalid-&g0.tab=1).

Reply:
Ah yep good point. Have sorted that.